multichain-testing/tools/hermes-tools.ts

```diff
@@ -0,0 +1,71 @@
+import type { IBCChannelID, IBCConnectionID, IBCPortID } from '@agoric/vats';
+import type { ExecSync } from './agd-lib.js';
+
+const kubectlBinary = 'kubectl';
+
+// based on config.yaml
+const relayerMap: { [key: string]: string } = {
+  osmosis: 'hermes-agoric-osmosis-0',
+  cosmoshub: 'hermes-agoric-gaia-0',
+};
+
+const getBinaryArgs = (chainName: string) => {
```

Comment:
good helper. more of a "make" than a "get". "binary" could be many things.
```suggestion
const makeKubeArgs = (chainName: string) => {
```